mappings/net/minecraft/client/font/BitmapFont.mapping

```diff
@@ -31,13 +23,13 @@ CLASS net/minecraft/class_386 net/minecraft/client/font/BitmapFont
 		METHOD method_51748 (Lcom/mojang/serialization/codecs/RecordCodecBuilder$Instance;)Lcom/mojang/datafixers/kinds/App;
 			ARG 0 instance
 		METHOD method_51749 validate (Lnet/minecraft/class_386$class_387;)Lcom/mojang/serialization/DataResult;
-			ARG 0 loader
+			ARG 0 fontLoader
 		METHOD method_51750 (Ljava/util/List;)[[I
-			ARG 0 strings
-		METHOD method_51752 validate ([[I)Lcom/mojang/serialization/DataResult;
-			ARG 0 codePointGrid
+			ARG 0 chars
```

Comment:
```suggestion
			ARG 0 strings
```
This is conversion of `List<String>` to `int[][]`, each `int[]` is array of codepoints